tests/test_utils.py

```diff
@@ -1,19 +1,14 @@
+import filecmp
 import os
+import shutil
+from pathlib import Path
 
 import numpy as np
 import pytest
 from osgeo import gdal
 
-from hyp3_isce2.utils import (
-    ESA_HOST,
-    GDALConfigManager,
-    extent_from_geotransform,
-    get_esa_credentials,
-    make_browse_image,
-    oldest_granule_first,
-    resample_to_radar,
-    utm_from_lon_lat,
-)
+from hyp3_isce2.utils import *
```

Comment:
```suggestion
import hyp3_isce2.utils as utils
```
This is better practice and Flake8 will not pass without this change. Unfortunately, you will have to update the references to utils throughout the file.